ast2vec/cloning.py

```diff
@@ -153,22 +156,19 @@ def generate_repo_urls(inputs):
             else:
                 yield item
 
-    def _find_enry(self, linguist):
+    def find_linguist(self, linguist):
         if linguist is None:
-            if self._languages:
-                self._log.info("Looking for enry to work with provided languages.")
-                linguist = shutil.which("enry", path=os.getcwd())
-                if linguist is None:
-                    linguist = "enry"
-            else:
-                return None, False
+            linguist = shutil.which("enry", path=os.getcwd())
```

Comment:
Can you please improve this: `path=os.getcwd() + ":" + os.getenv("PATH", os.defpath)`
if failed, directly raise `FileNotFoundError`